Simplenote.xcodeproj/project.pbxproj

```diff
@@ -3055,6 +3132,108 @@
 			};
 			name = Release;
 		};
+		BA8CF2212BFD20780087F33D /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
+				ASSETCATALOG_COMPILER_GENERATE_SWIFT_ASSET_SYMBOL_EXTENSIONS = YES;
+				CLANG_ANALYZER_NONNULL = YES;
+				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES_AGGRESSIVE;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
+				CLANG_ENABLE_MODULES = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
+				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
+				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
+				CODE_SIGN_STYLE = Automatic;
+				CURRENT_PROJECT_VERSION = 1;
+				DEBUG_INFORMATION_FORMAT = dwarf;
+				DEVELOPMENT_TEAM = ZL97FBBQH7;
```

Comment:
You're right.  It is currently set to automatic signing and set to my personal team.  I have updated this PR to use the certificate that was created by infrastructure, but it won't work yet cause the naming is off.

@spencertransier can you please take a look at this and update the certificate/extension naming so that the extension signing is for `SimplenoteIntents` instead of `Intents` we can't use Intents because it conflicts with apples Intents framework.  Please feel free to add commits to this PR if needed.